resolwe/flow/tests/test_api.py

```diff
@@ -367,6 +370,60 @@ def test_change_collection(self):
         data.refresh_from_db()
         self.assertEqual(data.collection, self.collection)
 
+    def test_move_to_collection(self):
```

Comment:
Add a test to verify that the user is not able to move the data to a collection where he doesn't have permissions.